packages/components/src/navigation/index.js

```diff
@@ -83,30 +89,36 @@ const Navigation = ( { activeItemId, children, data, rootTitle } ) => {
 		);
 	};
 
+	const render = ( { className: animateClassName } ) => (
+		<div
+			className={ classnames(
+				'components-navigation__level',
+				animateClassName
+			) }
+		>
+			{ children( {
+				level,
+				NavigationBackButton,
+				parentLevel,
+			} ) }
+		</div>
+	);
+
 	return (
 		<Root className="components-navigation">
-			<Animate
-				key={ level.id }
-				type="slide-in"
-				options={ {
-					origin: isNavigatingBack ? 'right' : 'left',
-				} }
-			>
-				{ ( { className: animateClassName } ) => (
-					<div
-						className={ classnames(
-							'components-navigation__level',
-							animateClassName
-						) }
-					>
-						{ children( {
-							level,
-							NavigationBackButton,
-							parentLevel,
-						} ) }
-					</div>
-				) }
-			</Animate>
+			{ onMount ? (
+				render( {} )
+			) : (
+				<Animate
+					key={ level.id }
+					type="slide-in"
+					options={ {
+						origin: isNavigatingBack ? 'right' : 'left',
+					} }
+				>
+					{ render }
```

Comment:
Instead of creating a separate render method, should we just not pass the animation class if we're mounting?

`isMounting ? null : animateClassName`